packages/tools/client-debugger/client-debugger/src/data-visualization/Overview.md

```diff
@@ -2,16 +2,23 @@
 
 This directory contains a system for describing Fluid Distributed Data Structure (DDS) visuals in a declarative manner, and for communicating incremental updates to those visuals.
 
-This system is designed to be compatible with the message-passing-based approach taken by the debugger, such that consumers can process the visuals however they wish.
+This system is designed to be compatible with the message-passing-based approach taken by the debugger, such that consumers can process the visual descriptors however they wish.
 
 -   In particular, this enables tooling that may live in another process; e.g. Chromium Browser Extensions.
 
 ## The Flow
 
 For the purpose of demonstrating the intended usage flow of the system, we will be looking at it from the context of the debugger, rather than viewing this system in isolation.
 
-When initializing the debugger, the initializing consumer may optionally provide a root DDS(s) they wish to have visualized by the tooling.
-If they do not specify this, then the tooling will not generate any visuals.
+**Terms**:
+
+-   "Application": The client application, which initializes the "debugger".
+-   "Debugger": Represents the client-side debugger, which communicates application state updates via message passing to the "consumer".
+-   "Consumer": External tooling which consumes data from the "debugger" via message passing.
```

Comment:
I'd add a note about the browser extension as an example of a consumer to drive it home. I know it's a separate thing and people could decide to use this piece on their own, but from our standpoint I think we can have them reference each other and it's ok.

Reply:
Yeah, that's a good call. I don't mind calling out examples, so long as it's clear that's all they are. And I think it helps lend clarity here. Will update.